### DISCUSSION TITLE
please add 1.20 add antoher versions

### DISCUSSION THREAD

**Discussion by Mashutkat** [BleachDrinker420/BleachHack#513] [General]
lastest version 1.19.4

**Comment by vil** [comment]
Dear User,

Thank you for reaching out regarding the Minecraft mod client, BleachHack. I regret to inform you that BleachHack has been discontinued, and there won't be any future updates or versions released for it. The development team has ceased working on the project, and as a result, the client is no longer being actively maintained.

Discontinuing a project is not an easy decision, but sometimes it becomes necessary due to various reasons, such as changes in development priorities or the inability to allocate sufficient resources to continue supporting the project. Unfortunately, BleachHack falls into this category.

While it's understandable that you may have been looking forward to additional versions or updates, it's important to respect the decision made by the developers. In the absence of future updates, it's recommended to explore alternative mod clients or find other existing Minecraft modifications that may suit your needs.

There are numerous modding communities and platforms available where you can discover a wide range of Minecraft modifications created by different developers. These communities often have active discussions, tutorials, and resources that can help you find alternative options that align with your preferences and requirements.

I apologize for any disappointment this news may cause, but I hope you will understand the circumstances behind the discontinuation of BleachHack. If you have any further questions or need assistance with anything else, feel free to ask. I'm here to help!

Best regards,
Vili, member of BleachHack corp.

**Reply by Mashutkat** [reply]
thanks
